docs/HowTo/Backup/Backup.md

```diff
@@ -1,57 +1,53 @@
-description: Backing up and restoring Besu   
+description: Backing up and restoring Besu
 <!--- END of page meta data -->
 
-# Backups 
+# Backups
 
-In a decentralized blockchain, data is replicated on other nodes so it cannot be lost. However, backing up 
-configuration and data ensures a smoother recovery.  
+In a decentralized blockchain, data replicates between nodes so it is not lost, but backing up
+configuration and data ensures a smoother recovery from corrupted data or other failures.
```

Comment:
This is a pretty long sentence now - 29 words.  I'd probably break it into two but will leave it to you to decide.

Reply:
Yes. Updated.